frontend/client/src/real/java/de.amosproj3.ziofa.client/RustClient.kt

```diff
@@ -99,9 +104,15 @@ class RustClient(private val inner: uniffi.client.Client) : Client {
 
     override suspend fun getConfiguration(): Configuration = inner.getConfiguration().into()
 
-    override suspend fun setConfiguration(configuration: Configuration) =
+    override suspend fun setConfiguration(configuration: Configuration): UInt =
         inner.setConfiguration(configuration.into())
 
+    override suspend fun getOdexFiles(pid: UInt): Flow<StringResponse> =
+        inner.getOdexFilesFlow(pid).mapNotNull { it.into() }
```

Comment:
Maybe we could return strings straight away here

Reply:
Agreed, we should already do this on the rust side like with `Count` and `u32` from the example.

Reply:
> Agreed, we should already do this on the rust side like with Count and u32 from the example

I don't understand that. I can change it to sending them as a vec/list if being desired.

Reply:
@Mr-Kanister I just meant a Flow<String> instead of Flow<StringResponse> since this only has one string property anyways, the flow is fine, i oversaw the break in the getOdexFilesFlow and thought it would never complete

Reply:
Valid, doable.

Reply:
Done